docs/content/en/docs/intro-klt.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction to the Keptn Lifecycle Toolkit
+linktitle: Introduction to the Keptn Lifecycle Toolkit
+description: Understand the Keptn Lifecycle Toolkit
+weight: 05
+cascade:
+  github_subdir: "docs/content/en/docs"
+  path_base_for_github_subdir: "/content/en/docs-dev"
+---
+
+The Keptn Lifecycle Toolkit (KLT) makes your deployments observable,
+brings application-awareness to your Kubernetes cluster,
+and helps you reliably deliver your application with:
+
+* Pre-Deployment Tasks: e.g. checking for dependant services,
+  setting the cluster to be ready for the deployment, etc.
+* Pre-Deployment Evaluations: e.g. evaluate metrics
+  before your application gets deployed (e.g. layout of the cluster)
+* Post-Deployment Tasks: e.g. trigger a test,
+  trigger a deployment to another cluster, etc.
+* Post-Deployment Evaluations: e.g. evaluate the deployment,
+  evaluate the test results, etc.
+
+All of these things can be executed on a workload or on an application level,
+whereby a Keptn application is a collection of multiple workloads.
+
+## Compare Keptn Lifecycle Toolkit and Keptn LTS
+
+The Keptn Lifecycle Controller (KLT) is a Keptn subproject
+whose design reflects lessons we learned while developing Keptn LTS.
+KLT recognizes that tools such as Argo and Flux
+are very good at deploying applications
+so adds pre-deployment and post-deployment evaluations and actions.
+For many installations, this provides the functionality they need
+with much less complexity than the Keptn LTS project.
+
+Keptn LTS is a fully-incubated, long-term support release
+that can deploy applications on platforms other than Kubernetes
+and currently offers functionality that KLT does not currently have
+such as the abiity to accomodate complex scoring algorithms for SLO evaluations
```

Comment:
```suggestion
to accomodate complex scoring algorithms for SLO evaluations
```

Reply:
Your rewrite doesn't seem syntactically valid.  I'm not sure why you object to this but I tried rewriting that paragraph a little bit.  I'm not crazy about making it sound like these three features are the only differences between Keptn and KLT rather than leaving it open-ended but whatever...  See if this is acceptable.